### PR TITLE
Fixed gh pages deployment

### PR DESCRIPTION
**Pull request by BelfordZ**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ const remarkCodesandbox = require("remark-codesandbox");
 const config = {
   title: "MetaMask Docs",
   // tagline: '',
-  url: "https://your-docusaurus-test-site.com",
-  baseUrl: "/",
+  url: "https://metamask.github.io/",
+  baseUrl: "/", // overwritten in github action for staging / latest
```
Comment (ziad-saab) [inline docusaurus.config.js:12]: could this be `process.env.SOMETHING` instead? this way we wouldn't need a `sed`?

Comment (BelfordZ) [reply docusaurus.config.js:12]: yeah actually i think so

Comment (BelfordZ) [reply docusaurus.config.js:12]: 1 sec ill try it locally

Comment (BelfordZ) [reply docusaurus.config.js:12]: @ziad-saab I think this will work too: https://github.com/MetaMask/mm-docs-v2/pull/42/files

Comment (ziad-saab) [reply docusaurus.config.js:12]: yes!

**Pull request closed without merging**